src/server/api/lib/fakeservice.js

```diff
@@ -90,8 +91,24 @@ async function handleIncomingMessage(message) {
   return part.id;
 }
 
+async function buyNumbersInAreaCode(organization, areaCode, limit) {
+  const rows = _.times(limit, () => ({
+    organization_id: organization.id,
+    area_code: areaCode,
+    phone_number: `+1${areaCode}555FAKE`,
```

Comment:
doesn't this need to be different for each row? -- I do like the TEXT in the number to make it 'really broken' -- maybe 555FK{random two digits}?
Similar to this code:
https://github.com/MoveOnOrg/Spoke/blob/main/src/integrations/contact-loaders/test-fakedata/index.js#L128-L130

Nit: Aesthetically, we try to avoid using lodash, when a native JS pattern is ok.  _.chunk seems like a decent exception.  I'm on the edge here -- I think I'd favor a boring for loop, so people don't need to lookup what `_.times` does.